### PR TITLE
Support standalone and nullable types for true, false, and null.

### PR DESCRIPTION
**Pull request by kschatzle**

### CODE REVIEW

File: src/Prophecy/Doubler/Generator/Node/TypeNodeAbstract.php [hunk 1]
```diff
@@ -61,6 +61,7 @@ protected function getRealType(string $type): string
             case 'callable':
             case 'bool':
             case 'false':
+            case 'true':
```
Comment (stof) [inline src/Prophecy/Doubler/Generator/Node/TypeNodeAbstract.php:64]: true is not a builtin type before PHP 8.2, so it should not be in this branch. It needs a version check, as done for `mixed`

**Review by stof — CHANGES_REQUESTED** [review]

**Comment by kschatzle** [comment]
Question in general. Since true isn't a builtin type until PHP8.2, the code that is being mocked shouldn't have true as a standalone type. So how do choose what to prevent via versioning and what not?

File: src/Prophecy/Doubler/Generator/Node/TypeNodeAbstract.php (hunk 1 above)
Comment (kschatzle) [reply src/Prophecy/Doubler/Generator/Node/TypeNodeAbstract.php]: Done.

**Comment by ciaranmcnulty** [comment]
@kschatzle not sure what the question means really? You have the PHP version checks in place that look correct, to me at least

Can you rebase please?

**Comment by ciaranmcnulty** [comment]
ok I fixed the conflict and fixed an issue with true handling - thanks!

**Pull request merged**